### ISSUE TITLE
Why call OctreeLoader.load() twice in Potree.loadPointCloud()?

### ISSUE THREAD

**Issue by yuejiangfeiming** [potree/potree#1440]
As title descrips, for 'metadata.json', calling OctreeLoader.load() twice.I can't get the point.
![2024-06-20_15-51](https://github.com/potree/potree/assets/36955000/29538d97-5a1a-49cc-a357-15bd9b884af8)

**Comment by LucaTiedemann** [comment]
I think this is also leading to some duplicate calls further down the line:

![grafik](https://github.com/potree/potree/assets/119896195/86a19128-2995-4cb2-80cf-fbd8375a66ee)

**Comment by victorchrollo14** [comment]
The new format from potreeconverter 2.0, doesn't load the entire model, throws the below errors.

I have a 500mb octree.bin file, which doesn't seem to be streaming similar to the old format rather it loads the whole file multiple times, and now I've got like 20gb of data loaded in the browswer.

@m-schuetz. Is the new potree format not stable?

![image](https://github.com/potree/potree/assets/89346667/e0d39acb-149c-4bbd-9b92-6478f26c276a)

![image](https://github.com/potree/potree/assets/89346667/e0733029-4fe6-4402-b1a6-1997d310921d)

**Comment by manye31** [comment]
@victorchrollo14 did you find a solution to this `failed to load r` error? I am encountering it as well when hosting from my server. But using the same format and pointcloud, it works fine locally.

**Comment by victorchrollo14** [comment]
> @victorchrollo14 did you find a solution to this `failed to load r` error? I am encountering it as well when hosting from my server. But using the same format and pointcloud, it works fine locally.

Yep. Your server must support partial fetching, Using the range header. As potree requests a partial part of the octree.bin file in each request.

**Comment by manye31** [comment]
> > [@victorchrollo14](https://github.com/victorchrollo14) did you find a solution to this `failed to load r` error? I am encountering it as well when hosting from my server. But using the same format and pointcloud, it works fine locally.
> 
> Yep. Your server must support partial fetching, Using the range header. As potree requests a partial part of the octree.bin file in each request.

Brilliant.  Thank you!
